plugins/sasl/data/modules/Custom Module/MCDU/pages/500-DATA/503-irs-monitor.lua

```diff
@@ -16,12 +16,44 @@
 
 local THIS_PAGE = MCDU_Page:new({id=503})
 
+function THIS_PAGE:calc(mcdu_data, i)
+    self:set_line(mcdu_data, MCDU_LEFT, i, "<IRS" .. i, MCDU_LARGE, ECAM_WHITE)
+    if ADIRS_sys[i].ir_status == IR_STATUS_IN_ALIGN then
+      self:set_line(mcdu_data, MCDU_LEFT, i+1, " ALIGN TTN " .. ADIRS_sys[i]:get_align_ttn(), MCDU_SMALL, ECAM_GREEN)
+    elseif ADIRS_sys[i].ir_status == IR_STATUS_ALIGNED then
+      self:set_line(mcdu_data, MCDU_LEFT, i+1, string.format(" NAV   DRIFT  %.2fNM/H", ADIRS_sys[i].ir_drift), MCDU_SMALL, ECAM_GREEN)
+    end
+end
+
 function THIS_PAGE:render(mcdu_data)
     self:set_title(mcdu_data, "IRS MONITOR")
 
-    
-    self:set_line(mcdu_data, MCDU_LEFT, 4, "PAGE NOT YET IMPLEMENTED", MCDU_LARGE, ECAM_MAGENTA)
+    self:calc(mcdu_data, ADIRS_1)
+    self:calc(mcdu_data, ADIRS_2)
+    self:calc(mcdu_data, ADIRS_3)
+
+    if adirs_how_many_irs_in_align() > 0 then
```

Comment:
This condition is not correct: the field is displayed as long as any ADIRS is in ATT mode (even if already aligned, the pilot has to update the hdg). Use `*.adirs_switch_status == ADIRS_CONFIG_ATT` to check.